lib/radiator/directory/editor/editor.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Radiator.Directory.Editor.Editor do
+  @moduledoc """
+  Manipulation of data with the assumption that the user has
+  the :edit permission to the entity.
+  """
+  import Ecto.Query, warn: false
+
+  alias Ecto.Multi
+
+  alias Radiator.Repo
+
+  alias Radiator.Directory.{
+    Network
+  }
```

Comment:
Only use `{}` syntax when there are at least two entries? Otherwise it looks too verbose.